pkg/proxy/healthcheck/proxier_health.go

```diff
@@ -187,22 +214,23 @@ func (h healthzHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
 	} else {
 		metrics.ProxyHealthzTotal.WithLabelValues("200").Inc()
 		resp.WriteHeader(http.StatusOK)
-		// In older releases, the returned "lastUpdated" time indicated the last
+		// In older releases, the returned "lastUpdatedMap" time indicated the last
 		// time the proxier sync loop ran, even if nothing had changed. To
 		// preserve compatibility, we use the same semantics: the returned
-		// lastUpdated value is "recent" if the server is healthy. The kube-proxy
+		// lastUpdatedMap value is "recent" if the server is healthy. The kube-proxy
 		// metrics provide more detailed information.
 		lastUpdated = currentTime
 	}
-	fmt.Fprintf(resp, `{"lastUpdated": %q,"currentTime": %q, "nodeEligible": %v}`, lastUpdated, currentTime, nodeEligible)
+	fmt.Fprintf(resp, `{"lastUpdatedMap": %q,"currentTime": %q, "nodeEligible": %v}`, lastUpdated, currentTime, nodeEligible)
```

Comment:
just leave this as `lastUpdated`... 
likewise below

Reply:
apologies for this, will update.